maintainers/maintainer-list.nix

```diff
@@ -2908,6 +2908,12 @@
     githubId = 538538;
     name = "Bryan Richter";
   };
+  chriscoffee = {
```

Comment:
Split into a separated commit, no need for a PR.

Reply:
Removed commit in https://github.com/NixOS/nixpkgs/pull/241666/commits/0b1234570c99e34904a0120caef1cc4c4ce033d3 and opened PR https://github.com/NixOS/nixpkgs/pull/248514 separating out `maintainers` addition.

Reply:
I said there is no need for a pull request.

Reply:
Apologies, I misunderstood. They were originally split back in July but I squashed them. I'll fix now.

Thanks for your patience.